data/portals.csv

```diff
@@ -753,14 +753,16 @@ southafrica_municipal_data,"Municipal Data, South Africa",https://municipaldata.
-spain,datos.gob.es - Official Spain Government Portal,http://datos.gob.es,Aporta Initiative - Spanish Government,Government of Spain,05/11/2015,Government,"Datos.gob.es is the national portal that organises and manages the Public Information Catalogue of the General State Administration.
+spain,datos.gob.es - Official Spain Government Portal,http://datos.gob.es,Aporta Initiative - Spanish Government,Government of Spain,5/11/2015,Government,"Datos.gob.es is the national portal that organises and manages the Public Information Catalogue of the General State Administration.
 
 The portal also provides general information, educational material and news updates on reusing public sector information.",level.national eu-official spain,other,http://datos.gob.es/node/533?language=in,Spain,"40.0028028,-4.0026",ES,es,active,2012-06-25T08:15:21.404Z,Custom/in-house implementation,,Other,http://datos.gob.es/sites/default/files/catalogo.rdf
 ssc-open-data,Sunshine Coast Council Open Data,https://data.sunshinecoast.qld.gov.au/,Stephen Gates,Sunshine Coast Council,17/08/2015,Government,"The Sunshine Coast is in Queensland, Australia and the Open Data Portal allows you to interactively explore data, maps, visualisations and application developer tools. ",australia local,Creative Commons - Attribution,http://creativecommons.org/licenses/by/3.0/au/deed.en,"Sunshine Coast, Queensland,","-26.654828, 153.092978",AU,en,active,,Socrata,,Socrata Open Data API (SODA),
 statcentral-ireland,StatCentral.ie,http://www.statcentral.ie/,Central Statistics Office Ireland,Central Statistics Office Ireland,,,"The StatCentral portal provides standard documentation on recurring official statistics and links to where they can be found. It covers data produced by the Irish Central Statistics Office (CSO), as well as other government departments, agencies and other state bodies.",ireland level.national portal statistics stats thematic eu-official,other-closed,,Ireland,"52.865196,-7.9794599",IE,en,active,2011-10-27T14:41:54.027Z,,,,
 state_of_illinois,Illinois,http://data.illinois.gov/,State of Illinois,State of Illinois,,,Open Data Portal for the State of Illinois,ctic level.local unitedstates,,,"Illinois, United States","40.0796319,-89.4339809",US,en,active,2011-06-27T18:15:57.1Z,,,,
 state_of_nebraska,Nebraska,http://www.nebraska.gov/data/,State of Nebraska,State of Nebraska,,,State of Nebraska Open Data Catalog,ctic level.national unitedstates,,,"Nebraska, United States","41.7370229,-99.5873816",US,en,active,2011-06-27T18:16:01.326Z,,,,
-state_of_washington,Washington,http://data.wa.gov/,State of Washington,State of Washington,,,Public Open Data of the State of Washington,ctic unitedstates,,,"Washington, United States","47.2868352,-120.2126139",US,en,active,2011-06-27T18:16:05.479Z,,,,

```

Comment:
From the your commit's description It looks like these (763-735) are the important lines (state of WA, USA).

All other changes just alter date formatting and quotes around cells that don't really add anything meaningful. Could you please change your PR in order to include only the relevant changes to the file?

Reply:
@will-saunders, could you fix your PR to include only the addition of the state of Washington, and not changes to other lines in the file?

Reply:
I'm not sure I can do that correctly; due to restrictions on my use of state equipment it is unlikely I will be able to install node or test adequately.  Please feel free to disregard the pull request. Is there another way to suggest changes?

Reply:
Let me try to clarify:  I did not intend any changes to other records, but clearly they happened.  I have no way of running the code locally, so I don't know how my changes might affect other records -- apart from guessing based on the format of the file,   I know of a number of other open data portals in the state, and would be happy to contribute them if I can figure out a way to do so without affecting the other entries.

Reply:
Well, spreadsheet software do have a tendency to convert records to a csv dialect of their own preference, e.g.: adding / removing quotes, change date formats, etc. I think that may be the reason for other records being altered here.

When I just need to add some lines to a csv and don't want to mess with the rest of the file, what I usually do is:

1. make a copy of the csv,
2. open it in the spreadsheet editor (in my case, LibreOffice Calc),
3. delete all lines except the header,
4. insert the new lines,
5. save the csv,
6. open in a plain text editor (e.g. notepad, gedit),
7. copy the new lines and append them at the end of the file,
8. commit and send the PR

If you are unable to do something like this, them, when I have some time, I intend to take the lines you added and copy / paste them into a new commit to include them.

In any case, thank you for your contributions. If you have more, please send them.